src/reducers/index.js

```diff
@@ -47,11 +47,39 @@ const cardReducer = (state = initialState, action) => {
   switch (action.type) {
     case ADD_CARD:
       action.payload.id = payloadId++;
-      return Object.assign({}, state, { cards: [...state.cards, action.payload] });
-    // case SELECT_CARD:
-    //   break;
-    // case CLEAR_CARD:
-    //   break;
+      return Object.assign({}, state, { selectedCard: action.payload, cards: [...state.cards, action.payload] });
+    case SELECT_CARD:
+      return Object.assign({}, state, { selectedCard: action.payload });
+    case DELETE_CARD:
+      let deleteState = [];
+      state.cards.map(card => {
```

Comment:
Would it make more sense to just use `.filter` here to get the new array excluding the deleted element?